src/obr/OpenFOAM/case.py

```diff
@@ -246,11 +246,11 @@ def decomposePar(self, args={}):
 
         method = args["method"]
         if method == "simple":
```

Comment:
```suggestion
        numberSubDomains = int(args.get("numberOfSubdomains", 0))
        if method == "simple":
```
Small nitpicky suggestion: Consider getting the numSubDomains before and and just use the variable the in ll.249,253, and 266.